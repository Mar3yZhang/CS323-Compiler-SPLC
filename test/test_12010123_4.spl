// Error type A at Line 5: unknown lexeme &
int test_4()
{
    int i = 0;
    while (&i <= 10 ) 
    {
        i = i + 1;
    }
}