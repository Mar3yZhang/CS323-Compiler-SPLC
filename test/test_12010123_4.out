Error type A at Line 5: unknown lexeme &